.github/workflows/gcc-problems.yml

```diff
@@ -20,23 +20,37 @@ on:
 
 # See [Example Sharing Container Between Jobs](https://github.com/docker/build-push-action/issues/225)
 jobs:
+  path_filter:
+    runs-on: ubuntu-latest
+    # Map a step output to a job output
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@master
+        with:
+          # Only run the main job for changes including the following paths
+          paths: '["orc8r/gateway/c/**", "lte/gateway/c/**"]'
```

Comment:
you did not deleted the path filter on the github action trigger in this file

Reply:
@quentinDERORY - addressed this comment and pushed the changes on to the branch. Please review.